Source/SwiftLintFramework/Rules/Style/UnusedOptionalBindingRule.swift

```diff
@@ -38,56 +37,51 @@ public struct UnusedOptionalBindingRule: ASTRule, ConfigurationProviderRule {
             "}\n"),
             Example("if let ↓(_, _, _) = getOptionalTuple(), let bar = Foo.optionalValue {\n" +
             "}\n"),
-            Example("func foo() {\nif let ↓_ = bar {\n}\n"),
```

Comment:
Why was this one removed?

Reply:
it wasn't, I just removed the trailing comma

Reply:
Got it